core/src/main/java/com/microsoft/applicationinsights/internal/agent/CoreAgentNotificationsHandler.java

```diff
@@ -507,7 +527,7 @@ private StringBuilder fetchExplainQuery(String commandName, Object object) {
                 }
                 explainSB.deleteCharAt(explainSB.length() - 1);
             }
-        } catch (Throwable t) {
+        } catch (Throwable t) { // FIXME can this be SQLException?
```

Comment:
@littleaj this could be SQLException. But I would leave it to Throwabl /Exception for now. Might be some string operations can lead to other exceptions not sure.